service/src/main/resources/changelog/changesets/20230123_remove_target_workflows.yaml

```diff
@@ -0,0 +1,44 @@
+databaseChangeLog:
+  - changeSet:
+      id: "remove all target workflows"
+      author: kpierre
+      changes:
+        - delete:
+            tableName: run_set
+            where: run_set_name = 'Target workflow 2, run 1'
+        - delete:
+            tableName: method_version
+            where: method_version_description = 'First version of target workflow 2'
+        - delete:
+            tableName: method
+            where: name = 'Target workflow 2'
+
+        - delete:
+            tableName: run_set
+            where: run_set_name = 'Target workflow 1, run 1'
+        - delete:
+            tableName: method_version
+            where: method_version_description = 'First version of target workflow 1'
+        - delete:
+            tableName: method
+            where: name = 'Target Workflow 1'
+
+        - delete:
+            tableName: run_set
+            where: run_set_name = 'Target workflow 4, run 1'
+        - delete:
+            tableName: method_version
+            where: method_version_description = 'First version of target workflow 4'
+        - delete:
+            tableName: method
+            where: name = 'Target Workflow 4'
+
+        - delete:
+            tableName: run_set
+            where: run_set_name = 'Target workflow 3, run 1'
```

Comment:
In the future, I would probably do this on ID rather than name, because (1) it's definitely unique and (2) it's less likely to have a typo, but in this case, this looks fine

Reply:
That's what I was originally intending, but was getting an error about comparing `uuid to integer`. When I tried casting, it errored out saying that I couldn't do that either. Maybe there's a specific syntax that I could have used?